### ISSUE TITLE
separate shopping_list_test.js and shopping_list_item_test.js

### ISSUE THREAD

**Issue by theRemix** [edsperanto/devleague--behavior-driven-shopping-list#12]
ex:
```
  js/shopping_list_test.js
  js/shopping_list_item_test.js
```

relates to #11

**Comment by ed-scaped** [comment]
fixed

**Comment by theRemix** [comment]
✅ lgtm 

for future reference, on the commit that fixes this issue, in the commit message use https://help.github.com/articles/closing-issues-via-commit-messages/

```
[your normal commit message]

resolves #12
```
